canonical-websites/design.ubuntu.com
# Remove old font pages and update layout

## Done

 - Removed the subnav from the font page and all the old subsections.
 - Updated the link to the licence

## QA

 - `./run`
 - Go to [http://localhost:8011/font/](http://localhost:8011/font/)
 - See that there is no font subnav
 - See that the link "open licence" goes to a subpage of ubuntu.com
